assembly/draw-functions.test.ts

```diff
@@ -19,7 +17,7 @@ import {
 
 var ctx: CanvasRenderingContext2D;
 var grd: CanvasGradient;
-var img: Image;
+var img: Image | null = null;
```

Comment:
Why must this be null?

Reply:
This value gets recreated all the time, and it was erroring whenever I tried to compile it otherwise.